bcda/auth/alpha_test.go

```diff
@@ -101,78 +99,6 @@ func (s *AlphaAuthPluginTestSuite) TestDeleteClient() {
 	assert.Empty(s.T(), aco.AlphaSecret)
 }
 
-func (s *AlphaAuthPluginTestSuite) TestGenerateClientCredentials() {
-	r, err := s.p.GenerateClientCredentials("", 0)
-	assert.Equal(s.T(), auth.Credentials{}, r)
-	assert.NotNil(s.T(), err)
-
-	aco := models.ACO{
-		UUID: uuid.NewRandom(),
-		Name: "Gen Client Creds Test",
-	}
-	err = connections["TestGenerateClientCredentials"].Save(&aco).Error
-	assert.Nil(s.T(), err, "wtf? %v", err)
-	// we know that we use aco.UUID as the ClientID
-	clientID := aco.UUID.String()
-
-	r, err = s.p.GenerateClientCredentials(clientID, 720)
-	assert.Equal(s.T(), auth.Credentials{}, r)
-	assert.Contains(s.T(), err.Error(), "have a registered client")
-
-	// quick and dirty register client
-	aco.ClientID = aco.UUID.String()
-	err = connections["TestGenerateClientCredentials"].Save(&aco).Error
-	assert.Nil(s.T(), err, "wtf? %v", err)
-	email := fmt.Sprintf("%s@genclientcredstest.com", testUtils.RandomHexID())
-	user, err := models.CreateUser("Fake User", email, aco.UUID)
-	assert.Nil(s.T(), err, "wtf? %v", err)
-
-	r, err = s.p.GenerateClientCredentials(clientID, 0)
-	assert.NotNil(s.T(), r)
-	assert.Nil(s.T(), err)
-
-	connections["TestGenerateClientCredentials"].Delete(&user, &aco)
-}
-
-func (s *AlphaAuthPluginTestSuite) TestRevokeClientCredentials() {
-	acoID := uuid.NewRandom()
-	var aco = models.ACO{
-		UUID:     acoID,
-		Name:     "RevokeClientCredentials Test ACO",
-		ClientID: acoID.String(),
-	}
-	db := connections["TestRevokeClientCredentials"]
-	db.Save(&aco)
-
-	email := fmt.Sprintf("%s@revokeclientcredentialstest.com", testUtils.RandomHexID())
-	var user = models.User{
-		UUID:  uuid.NewRandom(),
-		Name:  "RevokeClientCredentials Test User",
-		Email: email,
-		ACO:   aco,
-		ACOID: aco.UUID,
-	}
-	db.Save(&user)
-
-	clientID := user.ACOID.String()
-	_, err := s.p.GenerateClientCredentials(clientID, 0)
-	if err != nil {
-		assert.FailNow(s.T(), fmt.Sprintf(`can't create client credentials for %s because %s`, user.ACOID.String(), err))
-	}
-
-	assert := assert.New(s.T())
-
-	err = s.p.RevokeClientCredentials(aco.ClientID)
```

Comment:
same idea here as above.  Just run this and verify that the error comes back.  That way if the code gets reverted by mistake we might catch it in this test.